frontend/src/functions/formatPhone.ts

```diff
@@ -18,6 +18,17 @@ export type PhoneString = string;
 export function formatPhone(phoneNumber: PhoneString) {
   const parsedNumber = parsePhoneNumber(phoneNumber);
 
+  // This only checks for valid length of the phone number.
+  // Ex: ('+12223333333') === true
+  if (!parsedNumber?.isPossible()) {
+    // Bug: Any country code with two digits will break
```

Comment:
Could be a helpful clarification here to indicate why this bug isn't fixed yet, and when it should be:

```suggestion
    // Bug: Any country code with two digits will break
    //         This is fine as long as we only support `+1`.
```

Reply:
comment updated